projects/ngx-openlayers/src/lib/sources/imagearcgisrest.component.ts

```diff
@@ -12,25 +14,25 @@ import { SourceComponent } from './source.component';
   providers: [{ provide: SourceComponent, useExisting: forwardRef(() => SourceImageArcGISRestComponent) }],
 })
 export class SourceImageArcGISRestComponent extends SourceComponent implements OnInit, OnChanges {
-  instance: source.ImageArcGISRest;
+  instance: ImageArcGISRest;
 
   @Input() projection: ProjectionLike | string;
   @Input() url: string;
-  @Input() attributions: Attribution[];
+  @Input() attributions: AttributionLike[];
   @Input() crossOrigin?: string;
-  @Input() imageLoadFunction?: ImageLoadFunctionType;
+  @Input() imageLoadFunction?: LoadFunction;
   @Input() params?: { [k: string]: any };
   @Input() ratio = 1;
```

Comment:
By default ratio is set to 1.5

https://openlayers.org/en/latest/apidoc/module-ol_source_ImageArcGISRest-ImageArcGISRest.html